src/main/java/com/sublinks/sublinksapi/api/lemmy/v3/privatemessage/controllers/PrivateMessageController.java

```diff
@@ -95,7 +97,8 @@ PrivateMessagesResponse list(@Valid final GetPrivateMessages getPrivateMessagesF
         .perPage(perPage)
         .privateMessageSortType(sortType)
         .person(sender)
-        .unreadOnly(getPrivateMessagesForm.unread_only().orElse(false))
+        .unreadOnly(getPrivateMessagesForm.unread_only()
+            .orElse(false))
```

Comment:
We really need a PR to just fix the formatting. There is so much in this PR that is just reformatting that it makes the PR look bigger than it actually is